packages/workspace/src/browser/workspace-service.ts

```diff
@@ -321,6 +337,7 @@ export class WorkspaceService implements FrontendApplicationContribution {
     close(): void {
         this._workspace = undefined;
         this._roots.length = 0;
+        window.location.hash = '';
```

Comment:
do we have to do this in the close() function?

also, i notice that `window.location.hash` gets updated in other use cases (line 390 & 402). could we have a helper function responsible for updating it ?